src/Response/Embeds/RedirectResponse.php

```diff
@@ -27,8 +27,34 @@
 
 class RedirectResponse extends AbstractResponse
 {
+    /**
+     * Children of the response
+     *
+     * @var array<string, array<string, mixed>>
+     */
+    protected $childs = [
+        'Data' => [
+            'class' => Item::class,
+            'array' => 'Item'
+        ],
+    ];
     /**
      * @var string
      */
     public $Url;
+
+    /**
+     * @var string
+     */
+    public $Method;
+
+    /**
+     * @var Item
+     */
+    public $Data;
+
+    /**
+     * @var string
+     */
+    public $FlowType;
```

Comment:
Aren't you over complicated the regular redirect response? `FlowType` is something you will never find anywhere else besides the processor API. Therefore you should be extending the response, not modifying it.